packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -70,12 +78,44 @@ export async function buildAction(projectDir: string, options: Options): Promise
   }
 }
 
-async function statusAction(projectDir: string): Promise<void> {
-  throw new Error('not implemented yet');
+async function statusAction(projectDir: string, options: StatusOptions): Promise<void> {
+  const user: User = await UserManager.ensureLoggedInAsync();
+  const { exp }: ProjectConfig = getConfig(projectDir);
+
+  const accountName = exp.owner || user.username;
+  const projectName = exp.slug;
+
+  const projectId = await ensureProjectExistsAsync(user, {
+    accountName,
+    projectName,
+  });
+
+  const client = ApiV2.clientForUser(user);
+
+  const spinner = ora().start('Fetching build history...');
+
+  let buildStatus: { builds?: BuildInfo[] };
+
+  try {
+    buildStatus = await client.getAsync(`projects/${projectId}/builds/status`, {
+      platform: options.platform !== 'all' ? options.platform : undefined,
+      buildId: options.buildId,
+      status: options.status,
```

Comment:
```suggestion
      ...(['android', 'ios'].includes(platform) ? { platform } : {}),
      ...(buildId ? { buildId } : {}),
      ...(status ? { status } : {}),
```
otherwise you will endup with `?platform=&buildId=&status=`